### DISCUSSION TITLE
[Feature Request] POI reverse geocoding

### DISCUSSION THREAD

**Discussion by hermesespinola** [immich-app/immich#14436] [Feature Request]
Foursquare just open-sourced their poi dataset. I wonder if anyone else thinks it'd be a good idea to utilize this for reverse geocoding, similar to how other photos app do it:
![image](https://github.com/user-attachments/assets/d0950036-60f5-4152-8e8b-6622d23f1bbf)

https://tech.marksblogg.com/foursquare-open-global-poi-dataset.html

**Comment by zackpollard** [comment]
That's certainly interesting. We have changes in mind for the reverse geocoding in general. I'm not sure if ingesting this quantity of POIs into people's databases is reasonable. Will have to investigate further after the initial improvements to reverse geocoding

**Comment by hermesespinola** [comment]
maybe it could be done as an optional feature, off by default, and when turned on it'd download the dataset from aws S3.

**Comment by bo0tzz** [comment]
The foursquare suggestion is great. The FR for POI support already exists in #11486, so I'll close this as a duplicate of that.
